staging/src/k8s.io/csi-translation-lib/plugins/aws_ebs.go

```diff
@@ -253,3 +259,90 @@ func KubernetesVolumeIDToEBSVolumeID(kubernetesID string) (string, error) {
 
 	return awsID, nil
 }
+
+// This code is copied over from gce_pd as is. However, we're passing our own getRegionFromZones implementation.
+// It might be worth moving this to a common place and pass that function as a parameter.
```

Comment:
I noted this here from your conversation, but I can definitely move this to a common place.

Reply:
Yeah if you can move this to a shared library that would be great! cc @Jiawei0227

Reply:
Yeah I think it would be good if we can combine this into a common place, something like

```
type regionParserFn([]string) (string, error)
func regionTopologyHandler(pv *v1.PersistentVolume, regionParser regionParserFn) error {...}
```

Reply:
Will do! Is `in_tree_volume.go` a good place for it?

Reply:
Yep. Thanks!